cmd/snap/cmd_info.go

```diff
@@ -151,6 +152,15 @@ func (x *infoCmd) Execute([]string) error {
 
 		fmt.Fprintf(w, "name:\t%s\n", both.Name)
 		fmt.Fprintf(w, "summary:\t%q\n", both.Summary)
+		fmt.Fprintf(w, "description:\t%q\n", both.Description)
+		if len(both.Apps) > 0 {
+			apps := make([]string, len(both.Apps))
+			for i, a := range both.Apps {
+				apps[i] = a.Name
+			}
+			fmt.Fprintf(w, "apps:\t[%s]\n", strutil.Quoted(apps))
```

Comment:
Since apps cannot have any nasty characters I was wondering why would we be quoting that. I don't mind them quoted, just curious.